python_modules/libraries/dagster-dbt/dagster_dbt/cli/solids.py

```diff
@@ -190,7 +205,8 @@ def dbt_cli_run(context) -> DbtCliOutput:
         warn_error=context.solid_config["warn-error"],
         ignore_handled_error=context.solid_config["ignore_handled_error"],
     )
-    run_results = parse_run_results(context.solid_config["project-dir"])
+    target_path = __dbt_target_path(context.solid_config["target"])
```

Comment:
I think this should be context.solid_config["target-path"]. Also I think you can get rid of the __dbt_target_path function and just give the config field a default of "target" (`default_value="target"`, https://docs.dagster.io/_apidocs/config#dagster.Field)

Reply:
That's a much cleaner implementation to use the `default_value` config. I didn't know about it. Thanks!